packages/create-waku/package.json

```diff
@@ -2,9 +2,35 @@
   "name": "create-waku",
   "version": "0.4.10",
   "author": "Daishi Kato",
+  "contributors": [
+    "Vasu Singh"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/dai-shi/waku.git"
   },
-  "bin": "./cli.js"
+  "bin": "./dist/cli.js",
+  "files": [
+    "src",
+    "dist",
+    "template"
+  ],
+  "type": "commonjs",
```

Comment:
Future TODO: make type=module and use swc instead of esbuild.

Reply:
Sure, I will take look into this.